resources/migrations/000_migrations.yaml

```diff
@@ -11246,6 +11246,54 @@ databaseChangeLog:
         - sql:
             sql: DELETE FROM permissions_group WHERE name = 'MetaBot'
 
+  - changeSet:
```

Comment:
this new table is very much like `CollectionPermissionRevision` or `PermissionRevision` with the only difference that we don't have the `after` column in this table, instead, we have `changes`.

Note: both `CollectionPermissionRevision` and `PermissionRevision` have `after` column but it actually store the `changes` not the `after` graph.